readme.md

```diff
@@ -174,22 +174,23 @@ import ky from 'ky';
 
 ##### retry
 
-Type: `object | number`<br>
-
+Type: `number | object`<br>
 Default:
-- `limit`: `2`
-- `methods`: `get` `put` `head` `delete` `options` `trace`
-- `statusCodes`: [`408`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) [`413`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413) [`429`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) [`500`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) [`502`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502) [`503`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) [`504`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504)
-- `maxRetryAfter`: `undefined`
-
-An object representing `limit`, `methods`, `statusCodes` and `maxRetryAfter` fields for maximum retry count, allowed methods, allowed status codes and maximum [`Retry-After`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) time.
+- retries: `2`
```

Comment:
I know there needs to be a `limit` and `retryFunction` property... I'll do a follow-up PR once https://github.com/sindresorhus/got/pull/830 is done